about.html

```diff
@@ -1,130 +1,96 @@
+			</div>
+
+			<div class="col-md-8">
+				<nav>
+					<div class="nav-item"><a href="#">About</a>
+						<ul class="sub-menu">
+							<li><a href="about.html">About Codice</a></li>
+							<li><a href="governance.html">Governance</a></li>
+							<li><a href="board.html">The Board</a></li>
+							<li><a href="roles.html">Roles</a></li>
+							<li><a href="bylaws.html">Bylaws</a></li>
+						</ul>
+					</div>
+					<div class="nav-item"><a href="#">Contributing</a>
+						<ul class="sub-menu">
+							<li><a href="contributing.html">How To Contribute</a></li>
+							<li><a href="agreements.html">Agreements</a></li>
+							<li><a href="proposal.html">Proposal Templates</a></li>
+							<li><a href="incubation.html">Incubation</a></li>
+							<li><a href="graduation.html">Incubation Graduation</a></li>
+						</ul>
+					</div>
+					<div class="nav-item"><a href="tools.html">Tools</a></div>
+				</nav>
+			</div>
+		</div>
+	</div>
 
-    </section>
-          
-    <aside id="sidebar">
-      <nav id='menu' class='menu'>
-  <ul>
-    <li class='selected first'>
-<a href="index.html">Home</a>    <ul class='clearfix'>
-      <li class='selected first last'>
-<a href="about.html">About</a></li>
-    </ul>
-</li>
-<li>
-<a href="governance.html">Governance</a>    <ul class='clearfix'>
-      <li class='first'>
-<a href="board.html">The Board</a></li>
-<li class='last'>
-<a href="roles.html">Roles</a></li>
-    </ul>
-</li>
-<li>
-<a href="contributing.html">Contributing</a>    <ul class='clearfix'>
-      <li class='first'>
-<a href="agreements.html">Agreements</a></li>
-<li class='last'>
-<a href="proposal.html">Proposal Template</a></li>
-    </ul>
-</li>
-<li>
-<a href="incubation.html">Incubation</a>    <ul class='clearfix'>
-      <li class='first last'>
-<a href="graduation.html">Graduation</a></li>
-    </ul>
-</li>
-<li>
-<a href="tools.html">Tools</a></li>
-<li class='last'>
-<a href="projects.html">Projects</a></li>
-  </ul>
-</nav>
+	<div class="container">
+		<div class="main-content-wrap">
+				<h2>About the Codice Foundation</h2>
+				<p>The origin of the Codice Foundation idea is a bit difficult to trace.  However, it's been in the back of many individuals minds and hidden in the activities of many programs and projects for a long time (e.g. ARPA).  In reality, the Codice Foundation is an sister to the active Mil-OSS Working Group that was formed in 2008.  As a result of many events and conversations the need for an Apache-like foundation that could serve the needs of federally-focused projects (and in particular in the Defense industry) was identified.  At the Mil-OSS WG4 in October of 2012, the plans for initiating Codice began to unfold.  Also, with the transitioning of the Ozone Widget Framework from closed GOTS to OSS the time was right to provide a broader scope to the Mil-OSS working group - and to provide tools and support to other projects who could benefit in the same ways.</p>
```

Comment:
Yes, and we can change the sentence to "As a result of many events and conversations the necessity for an Apache-like foundation..."